res/values-it/derpquest_strings.xml

```diff
@@ -30,14 +33,23 @@
     <string name="theming_title">Temi</string>
     <string name="animations_title">Animazioni</string>
     <string name="nav_bar_category_title">Barra di navigazione</string>
+    <string name="statusbar_notifi_cat">Notifiche</string>

```

Comment:
this string is defined twice. make sure you've git the right one

Reply:
prev definition in line 796

Reply:
The one in line 796 is wrong, can you delete it? Thank you

Reply:
Yes will delete and merge now :) Thanks again